cv_next/app/feed/components/CVItemRSC.tsx

```diff
@@ -1,44 +1,53 @@
-import Image from "next/image"
+import Image, { StaticImageData } from "next/image"
 import Link from "next/link"
 import generateImageUrl from "@/helpers/imageURLHelper"
 import Categories from "@/types/models/categories"
-import { ClientCvModel } from "@/types/models/cv"
+import { useState } from "react"
+import blankCvImg from "../../../public/images/blackCv.png";
 interface CVCardProps {
-  cv: ClientCvModel
+  cv: CvModel
 }
 
 export default function CVItemRSC({ cv }: CVCardProps) {
   const categoryLink = `/feed?category=${Categories.category[
-    cv.categoryID
+    cv.category_id
   ].toLowerCase()}`
-  const imageUrl = generateImageUrl(cv.documentLink)
-  const formattedDate = new Date(cv.uploadDate).toLocaleDateString("en-US")
+  const imageUrl = generateImageUrl(cv.document_link)
+  const formattedDate = new Date(cv.created_at).toLocaleDateString("en-US")
+
+  const [imgSrc, setImgSrc] = useState<StaticImageData|string>(imageUrl)
+
+  const handleImgError = (e: any) => {
+    e.stopPropagation()
+    setImgSrc(blankCvImg)
+  }
 
   return (
     <>
       <Image
         width={500}
         height={500 * 1.4142}
         className="w-full rounded-lg p-2"
-        src={imageUrl}
+        src={imgSrc}
+        onError={handleImgError}
```

Comment:
Note that you can use `placeholder` property, perhaps will be better

Reply:
Instead of onError?

Reply:
Yeah because in the current behavior there's an implicit use of client-side behavior (states). Basically all the `handleImgError` functionality is probably not needed and can be handled in Next-native behavior that'd support static render.

Reply:
Managed to work fine with plaiceholder, added that